packages/campaignclassic/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@adobe/react-native-aepcampaignclassic",
+  "version": "1.0.0-beta.1",
+  "description": "Adobe Experience Platform support for React Native apps.",
```

Comment:
Experience Platform Campaign Classic extension for Adobe Experience Platform Mobile SDK. Written and Supported by Adobe.